src/psyclone/tests/domain/lfric/lfric_extract_driver_creator_test.py

```diff
@@ -270,7 +270,11 @@ def test_lfric_driver_simple_test():
     with open(filename, "r", encoding='utf-8') as my_file:
         driver = my_file.read()
 
-    for line in ["call extract_psy_data%OpenRead('field', 'test')",
+    for line in ["if (ALLOCATED(psydata_filename)) then",
```

Comment:
Ah, so we generate the same code irrespective of whether DM is enabled or not - it's a compile-time thing?

Reply:
Yes, nothing changes in the driver - we just potentially read from different input file(s). We can discuss at some stage if we want 'more' MPI support (though that's difficult, we would need to store info about domain decomposition, ... tbh ... let's not go there unless we have a real urgent need :)  )